package.json

```diff
@@ -40,19 +40,19 @@
     "chai": "1.4",
     "esformatter-pipe-test": "file:test/pipe",
     "esformatter-test-plugin": "file:test/plugin",
-    "esprima": "^2.0.0",
     "jshint": "~2.3.0",
     "mocha": "https://github.com/millermedeiros/mocha/tarball/latest",
     "mockery": "^1.4.0"
   },
   "dependencies": {
     "debug": "^0.7.4",
+    "espree": "^1.12.3",
```

Comment:
before we used `esprima` on the tests to validate that the output was still valid JS. now that we depend on `espree` we need to use it as well on the validation.